sync/awsdatabricks.py

```diff
@@ -92,7 +92,7 @@ def create_cluster(config: dict) -> Response[str]:
     :return: cluster ID
     :rtype: Response[str]
     """
-    response = get_default_client().create_cluster(config)
+    response = dbx.get_default_client().create_cluster(config)
```

Comment:
Made this change to faciliate `@patch`ing in tests

Reply:
I think `@patch("sync.awsdatabricks.get_default_client")` would have worked.

It is nice to have that additional context telling us it's a databricks client, but that can be had by aliasing get_default_client in the import statement.

Reply:
Also, I started mocking a bit lower down: https://github.com/synccomputingcode/syncsparkpy/blob/main/tests/api/test_predictions.py

It gives us more coverage, and doesn't require a mock client.

Reply:
> I think @patch("sync.awsdatabricks.get_default_client") would have worked.

I did try this first, and it didn't work, and so after poking around a little bit I found this - https://stackoverflow.com/questions/20242862/why-python-mock-patch-doesnt-work. It's interesting that it seems to work for the tests in `test_awsemr`, though...

Reply:
I think I misunderstood your comment when I first read it. So the patch should target the imported name _in the module we're testing_, not in the `clients.databricks` module. That makes more sense

Reply:
And indeed, that does work 🎉  thanks for the suggestion!